imports/plugins/core/checkout/client/methods/cart.js

```diff
@@ -11,7 +11,7 @@ Meteor.methods({
   // Under consideration for deprecation and migrating other payment Packages
   // to payments-stripe style methods
   "cart/submitPayment": function (paymentMethod) {
-    check(paymentMethod, Reaction.Schemas.PaymentMethod);
```

Comment:
I think this is actually a pretty important check here that shouldn't just be switched to `Object`. If the `paymentMethod` doesn't have specific values it's going to fail hard upstream and we should catch it here.

Reply:
@zenweasel It might be possible to use the schema, but the problem I was having with using the schema in some places like this is that the payment method being passed is somewhat different from what ultimately goes into the DB, either because of default values or for some other reasons. This object is validated against the schema already as part of the update call on line 59, so it seemed like checking it here is a bit superfluous anyway.